pkg/networkaware/topologicalsort/topologicalsort_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/rand"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-	"k8s.io/client-go/kubernetes/scheme"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
```

Comment:
nit: revert this?

Reply:
the scheme name is conflict with the `var scheme` for init schemes, changed to keep consistence